weblate/templates/accounts/email-sent.html

```diff
@@ -27,19 +27,19 @@ <h4 class="panel-title">
   <div class="panel-body">
 
     {% if is_reset %}
-      <p>{% blocktrans %}If your e-mail address exists in our database, you will receive further password recovery instructions to your inbox.{% endblocktrans %}</p>
-      <p>{% blocktrans %}If the confirmation link expires before you use it, request a new one.{% endblocktrans %}</p>
+      <p>{% blocktrans %}Follow the password recovery instructions sent to your e-mail inbox.{% endblocktrans %}</p>
+      <p>{% blocktrans %}Request a new confirmation link if it expires.{% endblocktrans %}</p>
     {% elif is_remove %}
-      <p>{% blocktrans %}Shortly, you should receive an e-mail with a confirmation link. Click it to remove your account.{% endblocktrans %}</p>
-      <p>{% blocktrans %}If the confirmation link expires before you use it, request a new one.{% endblocktrans %}</p>
+      <p>{% blocktrans %}Remove your account by clicking the confirmation link in the e-mail sent to you.{% endblocktrans %}</p>
+      <p>{% blocktrans %}Request a new confirmation link if it expires.{% endblocktrans %}</p>
     {% else %}
-      <p>{% blocktrans %}Thank you for registering. Click the confirmation link you receive by e-mail to complete the registration.{% endblocktrans %}</p>
```

Comment:
"will receive" would be better, but still misses the point.
Registration is referenced two times. The following course of action is not clear.